legacy/src/main/java/ca/bc/gov/app/repository/ForestClientRepository.java

```diff
@@ -59,6 +60,53 @@ Flux<ForestClientEntity> findClientByIncorporationOrName(
   Flux<ForestClientEntity> matchBy(String companyName);
 
   Mono<ForestClientEntity> findByClientNumber(String clientNumber);
```

Comment:
A tip for you here, you can use a Projection instead of the entity [as it was done here on silva](https://github.com/bcgov/nr-silva/blob/main/backend/src/main/java/ca/bc/gov/restapi/results/oracle/repository/OpeningRepository.java#L282) making your life easier

Reply:
I didn't touch that existing code. My method returns a DTO.